### PR TITLE
Dellmsijka adding a file for Nashua, NH

### LINKED ISSUE #72: Add a file for dellmsijka hometown
#Adding file for Nashua, NH

- [x] visit SkyVenture
- [ ] try Pressed Cafe

### PR DESCRIPTION
**Pull request by dellmsijka**
This pull request resolves #72 

@svietry look! i'm checking stuff in, Mark will be happy 🥇

### CODE REVIEW

File: dellmsijka-nashua.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Nashua, NH
```
Comment (dellmsijka) [inline dellmsijka-nashua.md]: Nashua, NH, who would live there??

File: dellmsijka-nashua.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+more info
```
Comment (dellmsijka) [inline dellmsijka-nashua.md]: too vague

File: dellmsijka-nashua.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Nashua, NH
+
```
Comment (dellmsijka) [inline dellmsijka-nashua.md]: empty lines are nonsense

File: dellmsijka-nashua.md (hunk 1 above)
Comment (gsirsat) [inline dellmsijka-nashua.md]: @dellmsijka ... Hello, great job.

Comment (gsirsat) [inline dellmsijka-nashua.md]: awesome... you fixed the issue...

**Pull request merged**